samcli/commands/check/resources/Graph.py

```diff
@@ -0,0 +1,23 @@
+"""
+Class for graph. All data is stored in the graph directly, or within nodes that are stored in the graph
+"""
+from typing import Any
+from samcli.commands.check.resources.LambdaFunction import LambdaFunction
+
+
+class Graph:
+    def __init__(self):
+        self._entry_points = []
+        self._resources_to_analyze = []
+
+    def add_entry_point(self, node: LambdaFunction):
+        self._entry_points.append(node)
+
+    def get_entry_points(self) -> Any:
```

Comment:
Is there a specific type to use instead of `Any`?

Reply:
Same as above. pylint thinks it returns an `Any`, so I need to use that to prevent make-pr from failing. Still looking into this.